res/css/views/rooms/_EventTile.scss

```diff
@@ -373,8 +373,8 @@ $left-gutter: 64px;
 
 .mx_EventTile_e2eIcon_unknown {
```

Comment:
While we're here, let's consolidate the blocks, joining the selectors that use the same values. It looks like there's only normal or warning images, so we should only need two blocks.